cogs/archive.py

```diff
@@ -45,33 +45,34 @@ async def autocomplete_get_categories(ctx: TeXBotAutocompleteContext) -> set[dis
             return set()
 
         return {
-            discord.OptionChoice(name=category.name, value=str(category.id))
-            for category
-            in main_guild.categories
-            if category.permissions_for(interaction_user).is_superset(
+            discord.OptionChoice(name=channel.name, value=str(channel.id))
+            for channel
+            in main_guild.channels
```

Comment:
I would reccomend using `main_guild.text_channels` to ensure only text channels are archived. Archiving other channel types is out of scope for this function